RELEASE-NOTES.md

```diff
@@ -1,3 +1,146 @@
 # Release notes for Reactive Streams
 
-Changes will be listed below after version 1.0.0 is released.
\ No newline at end of file
+---
+
+# Version 1.0.1 released on YYYY-MM-DD
+
+After more than two years since 1.0.0, we are proud to announce the immediate availability of `Reactive Streams version 1.0.1-RC1`.
+
+## Highlights:
+
+- Specification
+  + A new [Glossary](https://github.com/reactive-streams/reactive-streams-jvm/blob/v1.0.1-RC1/README.md#glossary) section
+  + Description of the intent behind every single rule
+  + No breaking semantical changes
+  + Multiple rule [clarifications](#specification-clarifications)
+- Interfaces
+  + No changes
+  + Improved JavaDoc
+- TCK
+  + Improved coverage
+  + Improved JavaDoc
+  + Multiple test [alterations](#tck-alterations)
+- New contributors since 1.0.0
```

Comment:
weird style to be honest, usually one lists contributors who contributed to a given milestone/release, not only the new ones?